src/components/about.js

```diff
@@ -0,0 +1,14 @@
+import { Typography } from "@mui/material";
+// eslint-disable-next-line no-unused-vars
```

Comment:
NBD, but if we are going to be doing this all over should we just set this in a repo-wide eslintrc?